rtil/src/native/hud.rs

```diff
@@ -13,7 +14,11 @@ use crate::statics::Static;
 
 static AMYHUD: Lazy<Static<usize>> = Lazy::new(Static::new);
 
-pub struct AMyHud;
+#[repr(C)]
+pub struct AMyHud {
+    #[cfg(windows)] _pad: [u8; 0x2b8],
+    pub b_showhud: u8,
```

Comment:
This field should be called `bitfield`, as it stored more than just `bShowHUD`. It should have a comment with all bits' names.

This is also how it's done e.g. in `FActorSpawnParameters`:
```rust
struct FActorSpawnParameters {
    ...
    // bRemoteOwned, bNoFail, bDeferConstruction, bAllowDuringConstructionScript
    bitfield: u8,
    ...
}
```